examples-site/react/example-client.tsx

```diff
@@ -2,6 +2,24 @@ import type { ComponentType } from 'react'
 import { StrictMode } from 'react'
 import { hydrateRoot } from 'react-dom/client'
 
+const errorDom = (
+  title: string,
+  info?: string,
+) => (
+  `<div class="govuk-error-summary" data-module="govuk-error-summary">
+  <div role="alert">
+    <h2 class="govuk-error-summary__title">${title}</h2>
+    ${
+    info && `<div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li>${info}</li>
+      </ul>
+    </div>`
+  }
+  </div>
+</div>`
+)
+
```

Comment:
I wanted to test the hydrationRoot function that works in all our examples 
 
![image](https://github.com/defencedigital/moduk-frontend/assets/5575331/547ab698-9a66-4114-8d7f-3a640cf6c669)
So I added error for our E2E test will fail if there is a mismatch